README.md

```diff
@@ -33,4 +33,46 @@ spec:
 to run ingress-node-firewall-operator unit tests (no cluster required), execute the following :
 ```shell
 make test
-```
\ No newline at end of file
+```
+
+## Running on a KinD cluster
+
+1. For fedora, you will need the following packages
+```sh
+sudo dnf install glibc-devel glibc-devel.i686
+```
+2. Download latest [KinD](https://kind.sigs.k8s.io/docs/user/quick-start) stable version
+3. Install KinD
+```sh
+./hack/kind-cluster.sh
+```
+4. Install custom resource definitions
+```sh
+make install
+```
+6. Build container images
+```sh
+make docker-build IMG=<some-registry>/ingress-node-firewall:tag
+```
+7. Load container image to KinD container(s)
+```sh
+kind load docker-image <some-registry>/ingress-node-firewall:tag
+```
+8. Deploy resources to KinD cluster
+```sh
+make deploy IMG=<some-registry>/ingress-node-firewall:tag
+```
+
+## Uninstall CRDs
+To delete the CRDs from the cluster:
+```sh
+make uninstall
+```
+## Undeploy controller
+UnDeploy the controller to the cluster:
+```sh
+make undeploy
+```
+
+## Disable webhook
+Remove manager binary flag `--enable-webhook` from the containers command in file config/manager/manager.yaml
```

Comment:
instead of doing this manually can we use a flag
```
ENABLE_VERIFICATION_WEBHOOK=false/true make deploy
```
then use that flag to edit the yaml with `yq` tool

Reply:
I think Ill handle this at a later date just to get this patch in and avoid rebase.